### ISSUE TITLE
https://noplayermusic.000webhostapp.co has been blocked by CORS policy [website is sleeping]

### ISSUE THREAD

**Issue by juanamm** [gsavio/player-shoutcast-html5#12]
Everything worked fine until a few hours ago.
Now the player has been blocked by CORS.
Not only on my site, but also on the @FabioTech site which was working fine: https://fabiotech.github.io/radio/

**Comment by gsavio** [comment]
Hi @juanamm, I did not understand your question very well, but maybe updating your project will solve your problem.

If you still need help, open a new issue, please.
